doc/en/api/1.0.0/gwcseed.yaml

```diff
@@ -0,0 +1,138 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: GeoWebCache Seeding and Truncating
+  description: A layer is a published resource (feature type or coverage).
+schemes:
+- http
+host: localhost:8080
+basePath: /geoserver/gwc/rest
+paths:
+  /seed.json:
+    get:
+      operationId: seedGet
+      summary: Query's currently running GWC task
+      description: Query's and returns a json array of the status for all currently running task. Requires .json extension in the request. The array contains a set of long in the following order:[tiles processed, total number of tiles to process, number of remaining tiles, Task ID, Task status].
+        The returned task status will be one of -1 = ABORTED, 0 = PENDING, 1 = RUNNING, 2 = DONE
+      produces:
+        - text/json
+      responses:
+        200:
+          description: OK
+          examples:
+            text/json: |
+              {"long-array-array":[
+                [364832,2761733810,998564,16,1],
+                [373472,2761733810,975448,17,1],
+                [363760,2761733810,1001509,18,1]
+              ]}
+  /seed/{layer}.{format}:
+    get:
+      operationId: layerSeedGet
+      summary: Query's the currently running GWC task for a given layer
+      description: Returns HTML UI page of running GWC task for a given layer and available parameters and tasks to execute orQuery's and returns a json array of the status for currently running task of a given layer if using json extension. The array contains a set of long in the following order:[tiles processed, total number of tiles to process, number of remaining tiles, Task ID, Task status]
+      produces:
+        - text/json
+        - application/html
+      parameters:
+        - name: layer
+          in: path
+          required: true
+          description: The name of the layer to query GWC task.
+          type: string
+        - name: format
+          in: path
+          required: true
+          description: Based on format, the request will return an application/html or text/json response.
+          type: string
+      responses:
+        200:
+          description: OK
+          examples:
+            text/json: |
+              {"long-array-array":[
+                [364832,2761733810,998564,16,1],
+                [373472,2761733810,975448,17,1],
+              ]}
+            application/html: |
+              <html>
+                <title>GWC Seed Form</title>
+                <body> {contents of GeoWebCache Seed Form} <body>
+              </html>
+        400:
+          description: Bad Request
+    post:
+      operationId: layerSeedPost
+      summary: Issue a seed, reseed or truncate task request
+      description: Executes an issue to seed, reseed or truncate task request for a layer and returns HTML UI page of running GWC tasks and tasks to execute.
+      consumes:
+        - text/json
+        - text/xml
```

Comment:
You have `text/xml` here, but `application/xml` elsewhere. Can you confirm that GWC really uses different media types for XML for different endpoints?